src/olympia/abuse/models.py

```diff
@@ -72,12 +75,42 @@ class AbuseReport(ModelBase):
         ('MANAGEMENT_WEBEXT_API', 5, 'Webext management API'),
         ('DRAG_AND_DROP', 6, 'Drag & Drop'),
         ('SIDELOAD', 7, 'Sideload'),
+        # Values between 8 and 13 are obsolete, we use to merge
+        # install source and method into addon_install_method before deciding
+        # to split the two like Firefox does, so these 6 values are only kept
+        # for backwards-compatibility with older reports and older versions of
+        # Firefox that still only submit that.
         ('FILE_URL', 8, 'File URL'),
         ('ENTERPRISE_POLICY', 9, 'Enterprise Policy'),
         ('DISTRIBUTION', 10, 'Included in build'),
         ('SYSTEM_ADDON', 11, 'System Add-on'),
         ('TEMPORARY_ADDON', 12, 'Temporary Add-on'),
         ('SYNC', 13, 'Sync'),
```

Comment:
Should these description strings be labelled as obsolete too? I.e. in the string itself.

Reply:
Yeah that's a good point, I'll do that.

Reply:
Actually, I think it's better to do that once Firefox 70 is released (with the patch from https://bugzilla.mozilla.org/show_bug.cgi?id=1572173) so that it's less confusing. I'll file an issue for us to update the strings later.